.github/workflows/deploy.yml

```diff
@@ -32,7 +32,6 @@ jobs:
           # Prevent use of implicit GitHub Actions read-only token GITHUB_TOKEN.
           # We push Javadocs to the gh-pages branch on commit.
           token: ${{ secrets.GH_TOKEN }}
-          fetch-depth: 0  # allow build-bin/idl_to_gh_pages to get the full history
```

Comment:
Didn't check the code just doublechecking this is fine the comment isn't about "license check"

Reply:
I should have self-commented. this repo doesn't have a script like this, so it is fuzz

Reply:
Well I guess it's copy pasta from a different repo :D

Reply:
yeah the dreaded zipkin-api repo ;)